BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonGroup.java

```diff
@@ -31,13 +32,15 @@ public BaragonGroup(@JsonProperty("name") String name,
                       @JsonProperty("sources") Set<String> sources,
                       @JsonProperty("defaultDomain") Optional<String> defaultDomain,
                       @JsonProperty("domains") Set<String> domains,
-                      @JsonProperty("domainAliases") Map<String, Set<String>> domainAliases) {
+                      @JsonProperty("domainAliases") Map<String, Set<String>> domainAliases,
+                      @JsonProperty(value = "minHealthyAgents", defaultValue = "1") int minHealthyAgents) {
```

Comment:
IDK which way we want to take this. 

On the one hand I think an additional configuration option makes some sense here; on the other hand I feel like in most cases we can probably safely calculate this based on the number of agents in the group.

IDK if we have that available but if we do I'd say we should only ever churn 75% of the agents in a group at a time?

Reply:
I kinda like the percentage idea, since it more easily applies across large and small groups